modules/core/src/main/scala/com.fortysevendeg.hood/csv/CsvService.scala

```diff
@@ -30,7 +30,12 @@ import scala.io.{BufferedSource, Source}
 
 trait CsvService[F[_]] {
 
-  def parseBenchmark(csvFile: File): F[Either[HoodError, List[Benchmark]]]
+  def parseBenchmark(
+      keyCol: String,
+      modeCol: String,
+      compareCol: String,
+      thresholdCol: String,
+      unitsCol: String)(csvFile: File): F[Either[HoodError, List[Benchmark]]]
```

Comment:
I'd create a config `case class` to group all these values, makes sense?

Reply:
@fedefernandez Agree... fixed!